src/main/java/com/java_academy/logic/model/Direction.java

```diff
@@ -0,0 +1,6 @@
+package com.java_academy.logic.model;
+
+public enum Direction {
+    HORIZONTAL,
+    VERTICAL
```

Comment:
I realized this yesterday, in requirements it's said that ship can be horizontal and vertical at the same time. So it can be shaped like a letter L or T or sth.